modules/core/src/main/java/org/apache/ignite/cache/affinity/Affinity.java

```diff
@@ -215,6 +231,7 @@
     public Map<Integer, ClusterNode> mapPartitionsToNodes(Collection<Integer> parts);
 
     /**
+     * @deprecated Use Affinity#mapPartitionToPrimaryAndBackupsList(int) instead
```

Comment:
This should be after @throws, at the end of the Javadoc.
Please, add a dot at the end of the statement.